code/SingleObjectAdmin.php

```diff
@@ -174,13 +174,13 @@ public function doSave($data, $form)
             }
         }
 
-        $link = '"' . $object->singular_name() . '"';
+        $link = '"' . $object->i18n_singular_name() . '"';
         $message = _t(
             'GridFieldDetailForm.Saved',
             'Saved {name} {link}',
             array(
-                'name' => $object->singular_name(),
-                'link' => $link
+                'name' => $object->i18n_singular_name(),
+                'link' => ''
```

Comment:
Why has this been removed? If we're removing it can we remove the `{link}` field and delete the array key?